changelog/16956.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+core: Load SSCT Token Generation counter from storage when upgrading a DR to a primary
```

Comment:
This describes the change but isn't meaningful to most changelog readers.  Can you instead describe the change from a user perspective?

Reply:
Done!